handsontable.d.ts

```diff
@@ -76,7 +77,7 @@ declare namespace _Handsontable {
     selectCellByProp(row: number, prop: string, endRow?: number, endProp?: string, scrollToCell?: boolean): boolean;
     setCellMeta(row: number, col: number, key: string, val: string): void;
     setCellMetaObject(row: number, col: number, prop: object): void;
-    setDataAtCell(row: number | any[], col: number, value: string | object, source?: string): void;
+    setDataAtCell(row: number | Array<[number,number,any]>, col: number, value: string | object, source?: string): void;
```

Comment:
I think this is better solved by #4818 using overloaded definitions. If an array is provided, the `col` and `value` parameters should _not_ be provided.